src/esp/metadata/attributes/AttributesBase.cpp

```diff
@@ -20,41 +32,85 @@ const std::map<std::string, SceneInstanceTranslationOrigin>
         {"com", SceneInstanceTranslationOrigin::COM},
 };
 
-std::string getShaderTypeName(int shaderTypeVal) {
-  if (shaderTypeVal <= static_cast<int>(ObjectInstanceShaderType::Unknown) ||
-      shaderTypeVal >=
-          static_cast<int>(ObjectInstanceShaderType::EndShaderType)) {
+// All keys must be lowercase
+const std::map<std::string, esp::physics::MotionType> MotionTypeNamesMap = {
+    {"static", esp::physics::MotionType::STATIC},
+    {"kinematic", esp::physics::MotionType::KINEMATIC},
+    {"dynamic", esp::physics::MotionType::DYNAMIC},
+};
+
+std::string getLightTypeName(esp::gfx::LightType lightTypeEnum) {
+  if (lightTypeEnum < esp::gfx::LightType::Point ||
```

Comment:
This seems a bit hacky. Please add a comment

Reply:
Ok

Reply:
Reading further I see that we pass `Unknown` intending to use the default. Maybe we should handle that explicitly and assert anything else.